scripts/run-pylint.sh

```diff
@@ -6,5 +6,11 @@ mkdir -p test/logs
 PYLINT_VIOLATIONS=test/logs/pylint.txt
 touch $PYLINT_VIOLATIONS
 
-pylint --django-settings-module=edx-ora2.settings.test --rcfile=pylintrc openassessment test --msg-template='"{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}"'> $PYLINT_VIOLATIONS
+export DJANGO_SETTINGS_MODULE=settings.test
+pylint --rcfile=pylintrc openassessment test --msg-template='"{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}"'> $PYLINT_VIOLATIONS
```

Comment:
changing the `--django-settings-module=` arg worked outside of the container, but not inside the container? I don't know. don't care. this works.

Reply:
Does this change anything about how/where we run the script? Does it need doc/README changes?

Reply:
Nope. This is just called straight, no args, from the makefile.